### PR TITLE
Incognito and download switch

### PR DESCRIPTION
**Pull request by Sadwhy**
This was a physical attack on me 😔

### CODE REVIEW

File: app/src/main/java/ani/dantotsu/download/manga/OfflineMangaFragment.kt [hunk 1]
```diff
@@ -177,8 +171,7 @@ class OfflineMangaFragment : Fragment(), OfflineMangaSearchListener {
                 snackString("no media found")
             }
         }
-        val total = view.findViewById<TextView>(R.id.total)
```
Comment (Sadwhy) [inline app/src/main/java/ani/dantotsu/download/manga/OfflineMangaFragment.kt:183]: Could you re add this before merging? I deleted it on accident.

File: app/src/main/java/ani/dantotsu/download/manga/OfflineMangaFragment.kt [hunk 2]
```diff
@@ -177,8 +171,7 @@ class OfflineMangaFragment : Fragment(), OfflineMangaSearchListener {
...
-        total.text = if (gridView.count > 0) "Manga and Novels (${gridView.count})" else "Empty List"
```
Comment (Sadwhy) [inline app/src/main/java/ani/dantotsu/download/manga/OfflineMangaFragment.kt:184]: Could you re add this before merging? Deleted on accident.

Comment (rebelonion) [reply app/src/main/java/ani/dantotsu/download/manga/OfflineMangaFragment.kt:184]: "this" is very specific

**Review by rebelonion — CHANGES_REQUESTED** [review]

File: app/src/main/java/ani/dantotsu/MainActivity.kt [hunk 3]
```diff
@@ -236,32 +243,12 @@ class MainActivity : AppCompatActivity() {
...
                                 )
                             }
-                            dismiss()
                         }
                     }.show(supportFragmentManager, "dialog")
                 }
             }
         }
 
-        GlobalScope.launch(Dispatchers.IO) {
```
Comment (rebelonion) [inline app/src/main/java/ani/dantotsu/MainActivity.kt:246]: why delete my logging?

**Review by rebelonion — CHANGES_REQUESTED** [review]

File: app/src/main/java/ani/dantotsu/MainActivity.kt (hunk 3 above)
Comment (Sadwhy) [reply app/src/main/java/ani/dantotsu/MainActivity.kt:246]: Probably because i had to test it

**Review by rebelonion — APPROVED** [review]

**Pull request merged**